### PR TITLE
tag develop when circle tests pass

### PR DESCRIPTION
**Pull request by MatthewBemis**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,9 +21,33 @@ jobs:
...
-          
+
+  tag:
+    docker:
+      - image: alpine/git
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "ab:2d:12:de:d1:ac:36:0b:6a:02:92:3f:2c:49:98:c9"
```
Comment (jacmrob) [inline .circleci/config.yml]: is this safe to be putting into source?  (i'm no ssh expert :( )

File: .circleci/config.yml [hunk 2]
```diff
@@ -21,9 +21,33 @@ jobs:
...
+      - checkout
+      - run:
+          when: on_success
+          command: |
+            echo 'export TAG_NAME=$(date +%s)' >> $BASH_ENV
```
Comment (jacmrob) [inline .circleci/config.yml]: does BASH_ENV come from the circle env vars, or are you expecting to pass it in somehow?

File: .circleci/config.yml [hunk 3]
```diff
@@ -21,9 +21,33 @@ jobs:
...
+            source $BASH_ENV
+            git --version
```
Comment (jacmrob) [inline .circleci/config.yml]: necessary?

File: .circleci/config.yml [hunk 4]
```diff
@@ -21,9 +21,33 @@ jobs:
...
+            git tag dev_tests_passed_$TAG_NAME
```
Comment (dvoet) [inline .circleci/config.yml]: put the dev_tests_passed_ prefix in the export statement above

File: .circleci/config.yml (hunk 1 above)
Comment (MatthewBemis) [reply .circleci/config.yml]: im just learning this now but the fingerprint is a hash of the public key so it's okay to publicize

File: .circleci/config.yml (hunk 2 above)
Comment (MatthewBemis) [reply .circleci/config.yml]: i think it's being created within the job. was following these instructions: https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command

**Pull request merged**